.github/actions/export-kbc-projects/action.yml

```diff
@@ -0,0 +1,29 @@
+  # Temporary removed project: https://keboolaglobal.slack.com/archives/C055HSMKX51/p1714024266416349
+#  { "host": "connection.north-europe.azure.keboola.com","project": 10890,"stagingStorage": "abs","backend": "snowflake","token": "$TEST_KBC_PROJECT_10890_TOKEN", "legacyTransformation": true },
+#  { "host": "connection.north-europe.azure.keboola.com","project": 10891,"stagingStorage": "abs","backend": "snowflake","token": "$TEST_KBC_PROJECT_10891_TOKEN", "legacyTransformation": true }
+#  { "host":"connection.north-europe.azure.keboola.com","project":10892,"stagingStorage":"abs","backend":"snowflake","token":"$TEST_KBC_PROJECT_10892_TOKEN", "legacyTransformation": true}
+#  {"host":"connection.north-europe.azure.keboola.com","project":10898,"stagingStorage":"abs","backend":"snowflake","token":"$TEST_KBC_PROJECT_10898_TOKEN", "legacyTransformation": true},
```

Comment:
Should we remove these projects? We have new ones, but the KBC team did fixes so the `workspace not found` error was probably erradicated from the tests results.

Reply:
You can enable them again, if they work, maybe some projects are already deleted cc @hosekpeter

Reply:
I don't know if the workspace delete operation is fixed, or the error doesn't occure more.

Reply:
I'll try it and we will see. If problem remains i will delete them forewer with secrets.